code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -226,10 +236,75 @@
 	else
 		return
 
+/mob/living/simple_animal/bot/medbot/proc/tip_over(mob/user)
+	playsound(src, 'sound/machines/warning-buzzer.ogg', 50)
+	user.visible_message("<span class='danger'>[user] tips over [src]!</span>", "<span class='danger'>You tip [src] over!</span>")
+	mode = BOT_TIPPED
+	var/matrix/mat = transform
+	transform = mat.Turn(180)
+	tipper_name = user.name
+
+/mob/living/simple_animal/bot/medbot/proc/set_right(mob/user)
+	if(user)
+		playsound(src, 'sound/machines/ping.ogg', 50)
+		user.visible_message("<span class='notice'>[user] sets [src] right-side up!</span>", "<span class='green'>You set [src] right-side up!</span>")
+		speak("Thank you!")
+	else
+		visible_message("<span class='notice'>[src] manages to writhe around enough to right itself.</span>")
+		tipper_name = null
+	tipped_status = MEDBOT_PANIC_NONE
+	mode = BOT_IDLE
+	transform = matrix()
+
+/// if someone tipped us over, check whether we should ask for help or just right ourselves eventually
+/mob/living/simple_animal/bot/medbot/proc/handle_panic()
+	tipped_status++
+	switch(tipped_status)

```

Comment:
I love this ID but the dialogue here seems like it could be workshopped to be more funny